nodejs/packagescanner.go

```diff
@@ -93,13 +93,17 @@ func (s *Scanner) Scan(ctx context.Context, layer *claircore.Layer) ([]*claircor
 	for _, p := range pkgs {
 		f, err := sys.Open(p)
 		if err != nil {
-			return nil, fmt.Errorf("nodejs: unable to open file: %w", err)
+			return nil, fmt.Errorf("nodejs: unable to open file %q: %w", p, err)
 		}
 
 		var pkgJSON packageJSON
 		err = json.NewDecoder(bufio.NewReader(f)).Decode(&pkgJSON)
 		if err != nil {
-			return nil, fmt.Errorf("nodejs: unable to decode package.json file: %w", err)
+			zlog.Warn(ctx).
```

Comment:
```suggestion
			zlog.Debug(ctx).
```
I don't think this warrants a warning.

If you want some logging that appears by default, this could collect up the paths of weird `package.json` files and emit an info log after this loop.

Reply:
I opted to collect the invalid paths and emit a warning log. If you think warning is wrong let me know